infra/build_deploy_config.py

```diff
@@ -89,8 +87,11 @@
     "GCP_DEFAULT_REGION",
 ]
 
+caller_info = boto3.client("sts").get_caller_identity()
+if '@' not in caller_info['UserId']:
+    raise ValueError('~/.aw/config needs to have an email under the role_session_name')
```

Comment:
Has `~/.aw/config` been verified to have an email on `gitlab` during automated deploy?  I know running this it errors on `dcp-developer` and I have to hack in my email.

Reply:
I removed this section of the code; i was having issues with 'assumed-role' w/ api 2fa. There will be other ways to enforce this, which is going to be handled with removed resources with invalid tags.